### PR TITLE
Constrain rpcport default values to a single location in code

### PR DESCRIPTION
**Pull request by luke-jr**

### CODE REVIEW

**Comment by pstratem** [comment]
concept ack

**Comment by MarcoFalke** [comment]
utACK bc40bbd

**Comment by sipa** [comment]
utACK

**Comment by jonasschnelli** [comment]
utACK

**Comment by paveljanik** [comment]
ACK (minor nit about formatting and & position ;-)

File: src/chainparamsbase.cpp [hunk 1]
```diff
@@ -86,18 +86,23 @@ const CBaseChainParams& BaseParams()
     return *pCurrentBaseParams;
 }
 
-void SelectBaseParams(const std::string& chain)
```
Comment (dcousens) [inline src/chainparamsbase.cpp]: Conflicts with https://github.com/bitcoin/bitcoin/pull/6907/files#diff-dbba8d0e1257cc14ce18de8fd6c9a0b1L89

**Comment by dcousens** [comment]
utACK

**Comment by petertodd** [comment]
utACK

**Pull request merged**